nextflow.config

```diff
@@ -63,6 +78,11 @@ if(!params.igenomesIgnore){
   includeConfig 'conf/igenomes.config'
 }
 
+// Avoid this error:
+// WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
+// Thanks to: https://github.com/alesssia/YAMP/wiki/How-to-use-Docker
+docker.runOptions = '-u $(id -u):$(id -g)'
```

Comment:
This should be added automatically using the template code. Please take this and add it to the main template on https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/nextflow.config